src/containers/People.js

```diff
@@ -95,7 +95,12 @@ export default class People extends React.Component {
       total, isMobile, dispatch, cacheId
     } = this.props
     const { currentUser } = this.context
-    if (error) return <AccessErrorMessage error={error}/>
+    if (error) return <AccessErrorMessage error={error} />
+
+    var { popularSkills } = community
```

Comment:
if you combine this and the next line as
```js
import { map } from 'lodash/fp'
const popularSkills = map(skill => skill.replace('-', '‑'), community.popularSkills)
```
then you won't have to create the empty array in the test. in general i think it's good to have the components be forgiving in what data they expect to receive.